src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py

```diff
@@ -1163,7 +1163,7 @@ def update_backup_schedule(cmd, resource_group_name, webapp_name, storage_accoun
     try:
         configuration = _generic_site_operation(cmd.cli_ctx, resource_group_name, webapp_name,
                                                 'get_backup_configuration', slot)
-    except CloudError:
+    except DefaultErrorResponseException:
```

Comment:
@Nking92, want to confirm this `DefaultErrorResponseException` does get thrown out by CLI?  I searched across the CLI, msrest but have not found the error definitions. Please clarify before i merge this.

Reply:
I just attached a debugger and inspected the type of exception getting thrown. It looks like this change in exception type came from the new Python SDK version. Maybe other CLI modules haven't had to handle the change in exception type yet.

Reply:
Got it, it is actually from [webapp's swagger](https://github.com/Azure/azure-rest-api-specs/blob/master/specification/web/resource-manager/Microsoft.Web/stable/2018-02-01/WebApps.json#L44). Don't want to cause trouble, but can your team please name it better or better just use CloudError?

Reply:
Glad to know where it came from. @naveedaz let's fix the error type.

Reply:
@yugangw-msft I talked to Navy about this change. He said the DefaultErrorResponse was made on behalf of a request by the ARM team. It's following a spec for the way errors should be returned. Is your concern mostly with the name of the exception? I think we want to keep the DefaultErrorResponseException now, because any change upstream will require a new API version, Swagger update, and SDK update

Reply:
@Nking92  we should get this change in this sprint, I am seeing errors locally when trying to run azwebapp up.

Reply:
@yugangw-msft CI is green, please merge if things look good to you